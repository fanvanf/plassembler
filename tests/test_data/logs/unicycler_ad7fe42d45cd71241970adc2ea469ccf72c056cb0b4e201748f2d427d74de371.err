Error: SPAdes encountered an error:

Command line: unicycler -1 tests/test_data/C11_subsetsim_R1.fastq -2 tests/test_data/C11_subsetsim_R2.fastq -l tests/test_data/chopper_long_reads.fastq.gz -t 1 -o tests/test_data/unicycler_output
