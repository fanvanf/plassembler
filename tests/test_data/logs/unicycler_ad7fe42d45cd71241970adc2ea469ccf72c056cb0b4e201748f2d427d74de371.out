
[93m[1m[4mStarting Unicycler[0m [2m(2023-05-27 13:09:50)[0m
[2m    Welcome to Unicycler, an assembly pipeline for bacterial genomes. Since you[0m
[2mprovided both short and long reads, Unicycler will perform a hybrid assembly.[0m
[2mIt will first use SPAdes to make a short-read assembly graph, and then it will[0m
[2muse various methods to scaffold that graph with the long reads.[0m
[2m    For more information, please see https://github.com/rrwick/Unicycler[0m

Command: [1m/Users/a1667917/mambaforge/envs/plassembler_env/bin/unicycler -1 tests/test_data/C11_subsetsim_R1.fastq -2 tests/test_data/C11_subsetsim_R2.fastq -l tests/test_data/chopper_long_reads.fastq.gz -t 1 -o tests/test_data/unicycler_output[0m

Unicycler version: v0.5.0
Using 1 thread

The output directory already exists and files may be reused or overwritten:
  /Users/a1667917/Documents/plassembler/tests/test_data/unicycler_output

Dependencies:
  [4mProgram       Version   Status[0m
  spades.py     3.15.2    [32mgood[0m  
  racon         1.5.0     [32mgood[0m  
  makeblastdb   2.12.0+   [32mgood[0m  
  tblastn       2.12.0+   [32mgood[0m  


[93m[1m[4mChoosing k-mer range for assembly[0m [2m(2023-05-27 13:09:52)[0m
[2m    Unicycler chooses a k-mer range for SPAdes based on the length of the input[0m
[2mreads. It uses a wide range of many k-mer sizes to maximise the chance of[0m
[2mfinding an ideal assembly.[0m

K-mer range already exists:
  /Users/a1667917/Documents/plassembler/tests/test_data/unicycler_output/spades_assembly/kmer_range

Will use this existing range:
  27, 53, 71, 87, 99, 111, 119, 127


[93m[1m[4mSPAdes assemblies[0m [2m(2023-05-27 13:09:52)[0m
[2m    Unicycler now uses SPAdes to assemble the short reads. It scores the[0m
[2massembly graph for each k-mer using the number of contigs (fewer is better) and[0m
[2mthe number of dead ends (fewer is better). The score function is 1/(c*(d+2)),[0m
[2mwhere c is the contig count and d is the dead end count.[0m

spades.py -o /Users/a1667917/Documents/plassembler/tests/test_data/unicycler_output/spades_assembly -k 27 --threads 1 --isolate -1 /Users/a1667917/Documents/plassembler/tests/test_data/C11_subsetsim_R1.fastq -2 /Users/a1667917/Documents/plassembler/tests/test_data/C11_subsetsim_R2.fastq -m 1024
